src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -32,15 +39,16 @@ export interface IAllRecordsState extends IPaginationBaseState {
   sortingOpened: boolean;
   filterOpened: boolean;
   isMapView: boolean;
+  recordViewType: 'GRID' | 'LIST';
   isRecordHighlighted: boolean;
   selectedRecord: string;
   selectedLat: number;
   selectedLng: number;
 }
 
 const withLatLong = (
-  wrappedComponent: string | ComponentClass<any> | StatelessComponent<any>
-): string | React.ComponentClass<any> | React.StatelessComponent<any> => wrappedComponent;
+  wrappedComponent: string | ComponentClass<any> | FunctionComponent<any>
```

Comment:
What is the purpose of this change?

Reply:
It was marked as deprecated

Reply:
Ok, we have three more usages of this, please change those too (filter-activity, filter-shelter, and shelter-details)